Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -622,7 +622,7 @@ class CombinatorialKalmanFilter {
                 boundParams.referenceSurface().getSharedPtr());
           }
 
-          stepper.releaseStepSize(state.stepping, ConstrainedStep::actor);
+          stepper.releaseStepSize(state.stepping, ConstrainedStep::navigator);
```

Comment:
So this I find a bit weird. Not saying that it made a ton of sense before, but in this case, we're unsetting the navigation constraint from the actor. This is a consequence of the renaming, so I understand this doesn't actually change the behavior (the navigator was setting the actor constraint before, if I'm not mistaken.).

Not sure what to do about this though.